common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```diff
@@ -31,6 +31,16 @@
     private static int[] completedOperations;
     private static long[] lastCompletionNanoTimes;
 
+    private static int getCompletedOperations() {
+        return IntStream.of(completedOperations).sum();
+    }
+
+    private static double getOperationsPerSecond() {
+        return IntStream.range(0, completedOperations.length)
+            .mapToDouble(i -> completedOperations[i] / (((double) lastCompletionNanoTimes[i]) / 1000000000))
```

Comment:
1000000000 can be extracted as a static constant.

Reply:
Fixed